server/container/util/src/test/java/org/apache/james/util/StreamUtilsTest.java

```diff
@@ -112,4 +116,41 @@ void ofNullableShouldReturnAStreamWithElementsOfTheArray() {
             .collect(Guavate.toImmutableList()))
             .containsExactly(1, 2);
     }
+
+    @Test
+    void ofEnumShouldReturnEmptyWhenEmptyEnum() {
+        assertThat(StreamUtils.ofEnum(newEnum())
+                .collect(Guavate.toImmutableList()))
+            .isEmpty();
+    }
+
+    @Test
+    void ofEnumShouldReturnValuesWhenFilledValues() {
+        assertThat(StreamUtils.ofEnum(newEnum(1, 2, 3))
+                .collect(Guavate.toImmutableList()))
```

Comment:
This collect is not needed: Use stream asserts! 

(and everywhere else in this class: this is some "relics from java8 migration for assertj)